packages/reason-relay/language-plugin/src/transformer/TypesTransformer.re

```diff
@@ -570,6 +575,21 @@ let getPrintedFullState =
   finalStr^;
 };
 
+let validStringLiteralRegex = [%bs.re "/^[_a-zA-Z][_a-zA-Z0-9]*$/"];
```

Comment:
this regex allows `_` as a valid string literal, which is not the case in OCaml, I think it should be this:
```suggestion
let validStringLiteralRegex = [%bs.re "/^(?:_[_a-zA-Z0-9]+|[a-zA-Z][_a-zA-Z0-9]*)$/"];
```

Reply:
Right! That makes sense. I've accounted for the single `_` case in the pattern match below, but including it in the regexp is probably better.